src/lib/notcurses.c

```diff
@@ -774,25 +774,24 @@ init_banner(const notcurses* nc){
 // practice is for the client code to have called setlocale() themselves, and
 // set the NCOPTION_INHIBIT_SETLOCALE flag. if that flag is set, we take the
 // locale as we get it.
-static void
-init_lang(const notcurses_options* opts){
-  if(!(opts->flags & NCOPTION_INHIBIT_SETLOCALE)){
-    const char* locale = setlocale(LC_ALL, "");
-    if(locale && (!strcmp(locale, "C") || !strcmp(locale, "POSIX"))){
-      const char* lang = getenv("LANG");
-      if(lang){
-        // if LANG was explicitly set to C/POSIX, roll with it
-        if(strcmp(locale, "C") && strcmp(locale, "POSIX")){
+void init_lang(int verbose){
```

Comment:
rather than this `verbose` flag, since (1) we protected logging against `NULL` nc for precisely the `ncdirect` path, and (2) we already have the loglevel initialized at this point for `notcurses`, just use `logwarn()`/`loginfo()` as appropriate.